apps/commudle-admin/src/app/feature-modules/users/components/public-profile/public-profile.component.html

```diff
@@ -12,12 +12,12 @@
                   <nb-icon
                     *ngIf="item.value.name !== 'Work History'"
                     [icon]="item.value.icon"
-                    [status]="item.value.icon_status"
+                    status="{{ highlight === item.value.link ? item.value.icon_status : 'default' }}"
                   ></nb-icon>
                   <fa-icon
                     *ngIf="item.value.name === 'Work History'"
                     [icon]="faBuilding"
-                    class="com-text-[#00D68F]"
+                    class="{{ highlight === item.value.link ? 'com-text-[#00D687]' : 'com-text-[#595867]' }}"
```

Comment:
try to add colour in a tailwind.preset files and use them here

Reply:
Okay